File: plugin/src/testKit/java/com/mobilesolutionworks/gradle/jacoco/jupiter/GradleRunnerExtensions.java
package com.mobilesolutionworks.gradle.jacoco.jupiter;

import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;

@Target(ElementType.PARAMETER)
@Retention(RetentionPolicy.RUNTIME)
public @interface GradleRunnerExtensions {

    Class<GradleRunnerWrapperExtension>[] values();
}

File: plugin/src/testKitKotlin/java/com/mobilesolutionworks/gradle/jacoco/jupiter/GradleRunnerExtensions.kt
package com.mobilesolutionworks.gradle.jacoco.jupiter

import kotlin.reflect.KClass

@Target(AnnotationTarget.VALUE_PARAMETER)
@Retention(AnnotationRetention.RUNTIME)
annotation class GradleRunnerExtensions(
        vararg val value: KClass<out GradleRunnerWrapperExtension>
)